api/tasks/vault.go

```diff
@@ -0,0 +1,50 @@
+package tasks
+
+import (
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	"github.com/ansible-semaphore/semaphore/db"
+	"github.com/ansible-semaphore/semaphore/util"
+)
+
+func (t *task) InstallVaultFiles() error {
+	var user db.User
+
+	if t.template.UserVault == true {
```

Comment:
you can simplify this to `if t.template.UserVault {`

Reply:
yes, that is a bad habit working with languages like PHP, where you need to do things like === to make sure, the boolean you have is actually a boolean because something like "if $somevar" behaves differently from "if $somevar === true". I'll go through the code and fix that.